# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/x86/microcode_ctl/no_install_rc.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2003-04-29 09:46:06.000000000 +0200
+++ ./Makefile	2003-04-29 09:46:22.000000000 +0200
@@ -50,18 +50,6 @@
 	$(INS) -m 644 $(MANPAGE) $(DESTDIR)$(MANDIR)
 	gzip -9f $(DESTDIR)$(MANDIR)/$(MANPAGE)
 
-	$(INS) -m 755 $(RCFILE) $(DESTDIR)$(RCFILETO)/$(RCFILEFINAL)
-
-ifndef DESTDIR
-		chkconfig --add $(RCFILEFINAL)
-else
-		echo "MAKE: Skipping chkconfig operation (rpm build?)"
-endif
-
-device:
-	mkdir -p $(DESTDIR)/dev/cpu
-	mknod $(DESTDIR)/dev/cpu/microcode c 10 184
-
 uninstall:
 ifndef DESTDIR
 	chkconfig --del $(RCFILEFINAL)
